src/vaadin-text-field-mixin.js

```diff
@@ -441,11 +435,7 @@
     // of the clear button instead of setting display property for it depending on state.
     /** @private */
     _stateChanged(disabled, readonly, clearButtonVisible, hasValue) {
-      if (!disabled &&
-          !readonly &&
-          clearButtonVisible &&
-          hasValue
-      ) {
+      if (!disabled && !readonly && clearButtonVisible && hasValue) {
```

Comment:
Remove IE11 workaround or adjust comment

Reply:
Removed the workaround.